File: print-manager-dotnet-core-cs/ResolutionExample.cs


using ceTe.DynamicPDF.Printing;

namespace print_manager_dotnet_core_cs
{
    internal class ResolutionExample
    {
        public static void Run()
        {
            PrintJob printJob = new PrintJob(Printer.Default, Util.GetPath("Resources/DocumentA.pdf"));
            printJob.PrintOptions.Resolution = ResolutionList.High;
            printJob.Print();
        }
    }
}

File: print-manager-dotnet-core-vb/ResolutionExample.vb
Imports ceTe.DynamicPDF.Printing

Friend Class ResolutionExample

        Public Shared Sub Run()

            Dim printJob As New PrintJob(
                Printer.Default,
                Util.GetPath("Resources/DocumentA.pdf"))

            printJob.PrintOptions.Resolution = ResolutionList.High

            printJob.Print()

        End Sub

    End Class
